packages/core/src/ceramic.ts

```diff
@@ -247,13 +264,14 @@ export class Ceramic implements CeramicApi {
       this._logger,
       this.dispatcher,
       modules.anchorService.validator,
-      this._streamHandlers,
-      this.context
+      this,
+      this._streamHandlers
     )
     const pinStore = modules.pinStoreFactory.createPinStore()
     const localIndex = new LocalIndexApi(
       params.indexingConfig,
-      this, // Circular dependency while core provided indexing
+      this,
+      this,
```

Comment:
Wow :) No need to fix it here. Just wow.

Reply:
Yep, was a good indicator about how much functionality was in this class.